server/services/store/sqlstore/licenses.go

```diff
@@ -0,0 +1,10 @@
+package sqlstore
+
+import (
+	sq "github.com/Masterminds/squirrel"
+	mmModel "github.com/mattermost/mattermost/server/public/model"
+)
+
+func (s *SQLStore) getLicense(_ sq.BaseRunner) *mmModel.License {
+	return nil
```

Comment:
Shouldn't this be something like this?
```suggestion
	return s.servicesAPI.GetLicense()
```